packages/web-api/src/response/AdminTeamsSettingsInfoResponse.ts

```diff
@@ -28,6 +28,7 @@ export interface Team {
   default_channels?:  string[];
   is_verified?:       boolean;
   enterprise_domain?: string;
+  url?:               string;
```

Comment:
I don't see any of `enterprise_domain`, `is_verified` nor `url` described in the [`admin.teams.settings.info` API method](https://api.slack.com/methods/admin.teams.settings.info) example response. Should it be documented there too?

Reply:
Yes, we can improve it. Not only the missing fields but also the invalid type data like `"default_channels": "array"`